core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -97,74 +92,6 @@ object UserService {
       "roles/bigquery.jobUser" -> Set(s"group:${ownerGroupEmail.value}", s"group:${computeUserGroupEmail.value}")
     )
 
-  // TODO - once workspace migration is complete and there are no more v1 workspaces or v1 billing projects, we can remove this https://broadworkbench.atlassian.net/browse/CA-1118
-  def deleteGoogleProjectIfChild(projectName: RawlsBillingProjectName,
-                                 userInfoForSam: UserInfo,
-                                 gcsDAO: GoogleServicesDAO,
-                                 samDAO: SamDAO,
-                                 ctx: RawlsRequestContext,
-                                 deleteGoogleProjectWithGoogle: Boolean = true
-  )(implicit ex: ExecutionContext) = {
-    def rawlsCreatedGoogleProjectExists(projectId: GoogleProjectId) =
-      gcsDAO.getGoogleProject(projectId) transform {
-        case Success(_) => Success(true)
-        case Failure(e: HttpResponseException) if e.getStatusCode == 404 || e.getStatusCode == 403 =>
-          Success(
-            false
-          ) // Either the Google project doesn't exist, or we don't have access to it because Rawls didn't create it.
-        case Failure(t) => Failure(t)
-      }
-
-    def F = Applicative[Future]
-
-    def deleteResourcesInGoogle(projectId: GoogleProjectId) =
-      for {
-        _ <- deletePetsInProject(projectId, gcsDAO, samDAO, ctx)
-        _ <- F.whenA(deleteGoogleProjectWithGoogle)(gcsDAO.deleteV1Project(projectId))
-      } yield ()
-
-    val projectId = GoogleProjectId(projectName.value)
-    samDAO.listResourceChildren(SamResourceTypeNames.billingProject,
-                                projectName.value,
-                                ctx.copy(userInfo = userInfoForSam)
-    ) flatMap { resourceChildren =>
-      F.whenA(
-        resourceChildren contains SamFullyQualifiedResourceId(projectName.value,
-                                                              SamResourceTypeNames.googleProject.value
-        )
-      )(
-        for {
-          _ <- rawlsCreatedGoogleProjectExists(projectId).ifM(deleteResourcesInGoogle(projectId), F.unit)
-          _ <- samDAO.deleteResource(SamResourceTypeNames.googleProject,
-                                     projectName.value,
-                                     ctx.copy(userInfo = userInfoForSam)
-          )
-        } yield ()
-      )
-    }
-  }
-
-  private def deletePetsInProject(projectName: GoogleProjectId,
```

Comment:
It took awhile to convince myself that this wasn't needed, particularly since it was checked in some tests for the v2 billing functionality.
Eventually I figured out that it only happens when `samDAO.listResourceChildren` returns projects, which only happens for v1.
So the tests were basically ensuring backwards compatibility with v1 projects.